src/DwitTech.AccountService.Core/Services/EmailService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using DwitTech.AccountService.Core.Dtos;
+using DwitTech.AccountService.Core.Interfaces;
+using DwitTech.AccountService.Core.Models;
+using DwitTech.AccountService.Data.Entities;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Services
+{
+    public class EmailService : IEmailService
+    {
+
+        private readonly IConfiguration _configuration;
+        public EmailService(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        public async Task<bool> SendMailAsync(Email email) //TODO
```

Comment:
There is a sendmail method in ActivationService. Move that implementation here and reference it.